src/gluonts/zebras/_period.py

```diff
@@ -178,6 +178,10 @@ def __lt__(self, other: Period) -> bool:
     def to_numpy(self) -> np.datetime64:
         return self.data
 
+    @property
+    def shape(self) -> Tuple[int, ...]:
```

Comment:
Just wondering: why would a `Period` need a `shape`?

Reply:
It wraps a numpy type that also has a `shape` attribute. It's empty (`()`) though.